src/parser.js

```diff
@@ -676,21 +676,25 @@ define([
             return;
         }
 
+        var required = el.popAttr('required');
+
         var attrs = {
             relevantAttr: parseVellumAttrs(form, el, 'relevant'),
             calculateAttr: parseVellumAttrs(form, el, 'calculate'),
             constraintAttr: parseVellumAttrs(form, el, 'constraint'),
             constraintMsgAttr: lookForNamespaced(el, "constraintMsg"),
-            requiredAttr: parseBoolAttributeValue(el.popAttr('required')),
+            requiredAttr: parseBoolAttributeValue(required),
+            requiredCondition: (el.popAttr('requiredCondition') ||
+                                ['true()', 'false()'].indexOf(required) === -1 ? required : undefined),
```

Comment:
This is hard to follow. Would it be acceptable to use only the required attribute and the parsing algorithm I outlined above?

Reply:
See the comment above. I can clean this up and not use a ternary nested in an if statement if this route is acceptable